build.sbt

```diff
@@ -1,10 +1,29 @@
+import sbt.Keys.libraryDependencies
 
-crossScalaVersions := Seq("2.11.12", "2.12.12")
+lazy val CommonSettings = Seq(
```

Comment:
nit: lowercase (since it's a [Scala val](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods))